### ISSUE TITLE
Screenshot is of whatever was previously on the clipboard

### ISSUE THREAD

**Issue by PachowStudios** [kufii/CodeSnap#44]
Issue is simple. Happening on Windows 10.

1. Copy something to the clipboard
2. Take a snap of something else in the same file
3. The resulting snap will be of what you copied to your clipboard previously
4. Switch away from the CodeSnap tab and back again. The image will refresh to the correct one.

I've noticed this in other cases too where the image just doesn't load until I switch away and back to the tab.

**Comment by kufii** [comment]
are you waiting for the flash animation before pasting? The flash animation is the confirmation that the screenshot has been taken

**Comment by AndhikaWB** [comment]
It's a focus related thing. You need to click CodeSnap tab first before copying the image. Otherwise, it will copy something else from your current focused tab (in this case I think you selected nothing so it keeps the previous clipboard). And as mentioned by kufii before, you know the image is already copied when it flashes.

**Comment by LERUfic** [comment]
Yeah, this is also happening to me. Instead using the selected lines, codesnap uses my clipboard for the image. Is this intended behavior or a bug?
```
Version: 1.52.0
Commit: 940b5f4bb5fa47866a54529ed759d95d09ee80be
Date: 2020-12-10T22:46:53.673Z (1 day ago)
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```
![Dec-13-2020 04-32-57](https://user-images.githubusercontent.com/30289344/101995461-83777000-3cfc-11eb-912d-c4a79166a7a6.gif)

**Comment by MrLuje** [comment]
@kufii I agree that the flow from quick actions works as intended, as it open the tab and let you select the code afterward.

But as stated by @LERUfic, by highlighting a piece of code, right click and selecting CodeSnap, I would believe that it would generate a snap of the current code and not a previously copied code (as a user who have never used this extension 😉)

**Comment by zinface** [comment]
I found that some characters may make it impossible to generate screenshots.
For example: 
```
"%02x:%02x:%02x:%02x:%02x:%02x"
```
This is c language formatted string (mac address code)
